sdk/template/Azure.Template/Directory.Build.props

```diff
@@ -1,4 +1,4 @@
-﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project ToolsVersion="16.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Why change this?

Reply:
Is this not the tools version we use? It was necessary for the SLN to compile, so I assumed it would make sense to also be in the props file.

Reply:
We have 0 files that have `Project ToolsVersion="16.0"`

Reply:
Fixed.